packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -56,11 +56,11 @@ export const GRID_ROWS_SCROLL_END = 'scroll:rowEnd';
 export const GRID_COL_RESIZE_START = 'colResizing:start';
 export const GRID_COL_RESIZE_STOP = 'colResizing:stop';
 
-export const GRID_COL_REORDER_START = 'colReordering:dragStart';
-export const GRID_COL_REORDER_DRAG_OVER_HEADER = 'colReordering:dragOverHeader';
-export const GRID_COL_REORDER_DRAG_OVER = 'colReordering:dragOver';
-export const GRID_COL_REORDER_DRAG_ENTER = 'colReordering:dragEnter';
-export const GRID_COL_REORDER_STOP = 'colReordering:dragStop';
+export const GRID_COLUMN_REORDER_START = 'colReordering:dragStart';
+export const GRID_COLUMN_REORDER_DRAG_OVER_HEADER = 'colReordering:dragOverHeader';
+export const GRID_COLUMN_REORDER_DRAG_OVER = 'colReordering:dragOver';
+export const GRID_COLUMN_REORDER_DRAG_ENTER = 'colReordering:dragEnter';
+export const GRID_COLUMN_REORDER_STOP = 'colReordering:dragStop';
```

Comment:
Why `GRID_COLUMN_REORDER_STOP` is not named `GRID_COLUMN_REORDER_DRAG_END` like the previous ones?

Reply:
No particular reason, I guess we missed it. It should follow the same convention so you can rename it 😉.